pkg/util/net.go

```diff
@@ -52,3 +53,21 @@ func RetryInsecureHttpsGet(url string) int {
 
 	return status
 }
+
+func RetryTCPConnection(host string, port string) int {
+	status := 0
+	err := wait.Poll(5*time.Second, 120*time.Second, func() (bool, error) {
+		timeout := 3 * time.Second
+		_, err := tcpnet.DialTimeout("tcp", tcpnet.JoinHostPort("127.0.0.1", "8445"), timeout)
+
+		if err == nil {
+			status = 200
```

Comment:
200 is HTTP status. If this is for TCP port probe, returning true/false is good to go